sdk/identity/azure-identity/azure/identity/_credentials/browser.py

```diff
@@ -2,18 +2,13 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-import platform
-import socket
-import subprocess
-import webbrowser
 from typing import Dict
 from urllib.parse import urlparse
 
 from azure.core.exceptions import ClientAuthenticationError
 
-from .. import CredentialUnavailableError
 from .._constants import DEVELOPER_SIGN_ON_CLIENT_ID
-from .._internal import AuthCodeRedirectServer, InteractiveCredential, wrap_exceptions
```

Comment:
You probably want to also remove the AuthCodeRedirectServer from inside internal